framework/src/main/java/org/tron/core/db/Manager.java

```diff
@@ -852,8 +852,8 @@ private void switchFork(BlockCapsule newHead)
               newHead.getBlockId(), getDynamicPropertiesStore().getLatestBlockHeaderHash());
     } catch (NonCommonBlockException e) {
       logger.info(
-          "this is not the most recent common ancestor, need to remove all "
-              + "blocks in the fork chain.");
+          "this is not the most recent common ancestor, "
```

Comment:
useless style change.

Reply:
just want to start a new line after the comma, looks friendly